crates/uv/tests/pip_install_scenarios.rs

```diff
@@ -735,6 +735,76 @@ fn multiple_extras_required() {
     assert_installed(&context.venv, "c_502cbb59", "1.0.0", &context.temp_dir);
 }
 
+/// all-extras-required
+///
+/// Multiple optional dependencies are requested for the via an 'all' extra.
```

Comment:
```suggestion
/// Multiple optional dependencies are requested for the package via an 'all' extra.
```

Reply:
Thanks! This file is actually generated. If you want to fix it, it's sourced from https://github.com/zanieb/packse/blob/main/scenarios/extras.json

Reply:
https://github.com/zanieb/packse/pull/112